src/db.c

```diff
@@ -155,7 +155,8 @@ void dbgAssertKeysizesHist(redisDb *db) {
     }
     kvstoreIteratorReset(&kvs_it);
     for (int type = 0; type < OBJ_TYPE_BASIC_MAX; type++) {
-        volatile int64_t *keysizesHist = kvstoreGetMetadata(db->keys)->keysizes_hist[type];
+        kvstoreMetadata *kvstoreMeta = (kvstoreMetadata *)kvstoreGetMetadata(db->keys);
+        volatile int64_t *keysizesHist = kvstoreMeta->keysizes_hist[type];
```

Comment:
```suggestion
        kvstoreMetadata *meta = kvstoreGetMetadata(db->keys);
        volatile int64_t *keysizesHist = meta->keysizes_hist[type];
```
I think we should keep short lines when the context is clear and nearby

Reply:
Done.